Cargo.toml

```diff
@@ -21,3 +21,7 @@ codegen-units = 1
 [patch.crates-io]
 zcash_encoding = { path = "components/zcash_encoding" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
+bellman = { git = "https://github.com/zkcrypto/bellman.git", rev = "29c82c7e432c1b32a50ef58e42790c3963ef2b4b" }
+orchard = { git = "https://github.com/zcash/orchard.git", rev = "61a67f018f456da980de7f93a5ea2a1dc5a61e63" }
```

Comment:
I verified that this commit includes the `ff 0.13` changes and is just before the `halo2_proofs` bump with the WASM fix.